source/documentation/index.md

```diff
@@ -20,10 +20,18 @@ The API follows REST principles and has a single POST method endpoint that retur
 
 EORI stands for Economic Operators Registration and Identification. It is a unique identification number used by customs authorities across the European Union (EU) to track imports and exports. Any business or individual shipping goods internationally will have an EORI number. This API uses this number to check the authorisation of NOP Waivers. This system replaced the older Trader's Unique Reference Number (TURN) system. For those in the UK, HM Revenue and Customs (HMRC) issues these numbers.
 
-A breakdown of the EORI number format for UK VAT-registered businesses:
-- GB: Indicates that the business is based in the UK.
+A typical EORI number for UK VAT-registered businesses is as follows:
+
+```python
```

Comment:
i don't think you need python here. You can just omit that word. or put **text** instead.